packages/server/src/clients/service.ts

```diff
@@ -56,7 +56,7 @@ export class ClientService extends Service {
     return client
   }
 
-  async getById(id: uuid): Promise<Client | undefined> {
+  async fetchById(id: uuid): Promise<Client | undefined> {
```

Comment:
Private?

Reply:
The fetch function can and is often used outside the service. It's not a public/private thing. The difference between the get and fetch function is intent. With the get function you indicate that the result must not be null, whereas with the fetch you indicate that you will handle the null value yourself.